### ISSUE TITLE
Naming conventions for adsorbates

### ISSUE THREAD

**Issue by rwest** [cfgoldsmith/RMG-Py#65]
In Slack, @cfgoldsmith wrote:
> Hi all. Let’s start a discussion on naming conventions for the Pt species. For example, we could adopt the strategy everyone uses, such as `CH3*`. Or we could put the * on the bonding atom (e.g. either `C*H3` or `H3C*`). This becomes more important for some of the more obscure species. What should we call `CNH`? Right now we have `CNH*`, but this obscures the fact that it is bonding through carbon and looks more like `HN=C=Pt`. So should we go with `HNC*`, `C*NH`, or what?

**Comment by rwest** [comment]
For not-adsorbed things, people put the Hn as close as possible to the C, but after it.
```
CH3OH 
CH3CH3
CH3OCH3
```

1. Put the * at the end of the group definition, after all other ligands?  **worst?**
```
CH3*
CNH*
CH2OH*
```

2. Put the * almost as close as possible to the C, but less close than the H's, and always after.
```
CH3*
C*NH
CH2*OH
CH2*CH2* (bidentate)
```

3. Put the * as close as possible to the C, even more close than the H's, but always after
```
C*H3
C*NH
C*H2OH
C*H2C*H2 (bidentate)
```

4. Can * come first? In which case could be to the left of the adatom
```
*CH3
*CNH
*CH2OH
*CH2*CH2  (bidentate)
```

I think my preference is 3 (or maybe 4 if starting with a * is never a problem?)
Emily prefers 4.

I notice that only the first one (which I think is worst) makes the current `CNH*` that you mention.

Is this a logical way of describing the options? Did I miss any? which do you like?

**Comment by cfgoldsmith** [comment]
I like 3 and 4 best as well. I just checked, and we can have a species in cantera start with *, but I don't know if it would work in Chemkin.

**Comment by kblondal** [comment]
I like 3 and 4 too. Say we use 4 in the paper, then what about physisorbed species like *(H2O), *(HNO), *(HCN) and *(HOOH)? Also put H after the binding atom, *(OH2), *(NHO), *(CHN), *(OHOH)?

**Comment by rwest** [comment]
I thought physisorbed species don't have a binding atom?

**Comment by rwest** [comment]
Also, for what it's worth, I found some CHEMKIN II instructions saying  "Species names may not begin with a number, a +, or an =, or have imbedded
blanks; an ionic species may end with any number of +'s or -'s; an imbedded
plus sign (+) must be enclosed in parentheses."

**Comment by kblondal** [comment]
Right, referring to it as binding atom because is probably not appropriate for vdW species. I mean the atom we classify the physisorbed species under in the thremo tree diagrams. Ok to have H in front for those?
